api/src/main/java/org/pmiops/workbench/api/AppsController.java

```diff
@@ -66,14 +66,6 @@ public ResponseEntity<EmptyResponse> deleteApp(
     return ResponseEntity.ok(new EmptyResponse());
   }
 
-  @Override
-  public ResponseEntity<UserAppEnvironment> getApp(String workspaceNamespace, String appName) {
```

Comment:
this is unused.  The UI never asks for an app by name.  Instead we always list all apps in a workspace.